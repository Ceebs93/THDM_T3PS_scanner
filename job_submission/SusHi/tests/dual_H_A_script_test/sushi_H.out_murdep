# mh =   0.71385800E+03 GeV
# muR/GeV - sigma(LO)/pb - sigma(NLO)/pb- sigma(NNLO)/pb - sigma(N3LO)/pb
   35.692899999999995       0.91461309641359001        1.0532099917777580       0.72477001211751624        0.0000000000000000     
   37.375054552725935       0.89841927255753695        1.0477764092797988       0.73947071305370260        0.0000000000000000     
   39.136486607119082       0.88265175526372952        1.0421956003016464       0.75328502560276189        0.0000000000000000     
   40.980932396726089       0.86729571130147631        1.0364814714161263       0.76625592466694314        0.0000000000000000     
   42.912304238355851       0.85233694463218979        1.0306469885623692       0.77842416834365213        0.0000000000000000     
   44.934698830628015       0.83776187448327011        1.0247042481038569       0.78982842482895910        0.0000000000000000     
   47.052405943619981       0.82355748591949918        1.0186645238410588       0.80050538841212227        0.0000000000000000     
   49.269917518044316       0.80971131485336856        1.0125383320306953       0.81048989305388797        0.0000000000000000     
   51.591937193257294       0.79621141669993789        1.0063354796618036       0.81981501563957604        0.0000000000000000     
   54.023390284308725       0.78304634061074174        1.0000651119664337       0.82851217363439567        0.0000000000000000     
   56.569434229195281       0.77020510518492002       0.99373575654029045       0.83661121673233163        0.0000000000000000     
   59.235469528478106       0.75767717556245873       0.98735536433582771       0.84414051290549530        0.0000000000000000     
   62.027151200468566       0.74545244181073622       0.98093134776783542       0.85112702923062677        0.0000000000000000     
   64.950400776280247       0.73352119852174880       0.97447061615128994       0.85759640784168378        0.0000000000000000     
   68.011418860190403       0.72187412554316821       0.96797960867313182       0.86357303733198898        0.0000000000000000     
   71.216698281952731       0.71050226977180353       0.96146432508301594       0.86908011990591838        0.0000000000000000     
   74.573037868960128       0.69939702794304304       0.95493035427306494       0.87413973455850846        0.0000000000000000     
   78.087556867468962       0.68855013035449164       0.94838290090296939       0.87877289654140633        0.0000000000000000     
   81.767710043474807       0.67795362546628335       0.94182681021422221       0.88299961335525423        0.0000000000000000     
   85.621303495270709       0.66759986532451088       0.93526659116586575       0.88683893749161036        0.0000000000000000     
   89.656511211228292       0.65748149175787585       0.92870643801369557       0.89030901613186386        0.0000000000000000     
   93.881892407922649       0.64759142330102570       0.92215025044526744       0.89342713799614426        0.0000000000000000     
   98.306409685378796       0.63792284280118228       0.91560165237433133       0.89620977752181785        0.0000000000000000     
   102.93944803794757       0.62846918566756338       0.90906400949031285       0.89867263653886909        0.0000000000000000     
   107.79083476113713       0.61922412872576860       0.90254044565105462       0.90083068359793894        0.0000000000000000     
   112.87086029662393       0.61018157964180397       0.89603385820037551       0.90269819109627591        0.0000000000000000     
   118.19030005965972       0.60133566688270523       0.88954693228573056       0.90428877033697574        0.0000000000000000     
   123.76043729517160       0.59268073018287115       0.88308215424565017       0.90561540464786283        0.0000000000000000     
   129.59308701103743       0.58421131148721606       0.87664182413136982       0.90669048067786462        0.0000000000000000     
   135.70062103930158       0.57592214634405536       0.87022806742227932       0.90752581798098442        0.0000000000000000     
   142.09599427848920       0.56780815572241028       0.86384284599040495       0.90813269699064392        0.0000000000000000     
   148.79277217268330       0.55986443822996357       0.85748796836508556       0.90852188548049750        0.0000000000000000     
   155.80515948565011       0.55208626270940808       0.85116509934523488       0.90870366360152555        0.0000000000000000     
   163.14803043104774       0.54446906119230776       0.84487576900316719       0.90868784757939292        0.0000000000000000     
   170.83696022262697       0.53700842219087996       0.83862138111601925       0.90848381214115936        0.0000000000000000     
   178.88825811134862       0.52970008430697901       0.83240322107648745       0.90810051178497952        0.0000000000000000     
   187.31900197949099       0.52253993014683942       0.82622246332005167       0.90754650090405264        0.0000000000000000     
   196.14707456512795       0.51552398053120685       0.82008017825053947       0.90682995285029466        0.0000000000000000     
   205.39120139381410       0.50864838895079501       0.81397733876681799       0.90595867802256025        0.0000000000000000     
   215.07099049793442       0.50190943629820861       0.80791482637752465       0.90494014101172593        0.0000000000000000     
   225.20697400797081       0.49530352583766057       0.80189343693854231       0.90378147686186605        0.0000000000000000     
   235.82065170390311       0.48882717840487988       0.79591388603768476       0.90248950649737203        0.0000000000000000     
   246.93453661912514       0.48247702782524443       0.78997681404930653       0.90107075136283898        0.0000000000000000     
   258.57220279360615       0.47624981653888060       0.78408279088001154       0.89953144731964463        0.0000000000000000     
   270.75833527759147       0.47014239142211295       0.77823232042518875       0.89787755784048440        0.0000000000000000     
   283.51878249190298       0.46415169979525966       0.77242584475474918       0.89611478654064114        0.0000000000000000     
   296.88061105590509       0.45827478560733664       0.76666374804520898       0.89424858908241123        0.0000000000000000     
   310.87216319943366       0.45250878578876791       0.76094636027408602       0.89228418448694891        0.0000000000000000     
   325.52311688046518       0.44685092676370125       0.75527396069154029       0.89022656588568794        0.0000000000000000     
   340.86454873604470       0.44129852111345713       0.74964678108203042       0.88808051074053873        0.0000000000000000     
   356.92899999999986       0.43584896438739917       0.74406500883692028       0.88585059057068416        0.0000000000000000     
   373.75054552725959       0.43049973204457898       0.73852878982976455       0.88354118018784300        0.0000000000000000     
   391.36486607119093       0.42524837653098374       0.73303823113125488       0.88115646649728818        0.0000000000000000     
   409.80932396726081       0.42009252447793305       0.72759340355444480       0.87870045686313591        0.0000000000000000     
   429.12304238355841       0.41502987402336189       0.72219434405951477       0.87617698708484160        0.0000000000000000     
   449.34698830628048       0.41005819223958734       0.71684105800290043       0.87358972897584597        0.0000000000000000     
   470.52405943619993       0.40517531267128604       0.71153352125907465       0.87094219758773128        0.0000000000000000     
   492.69917518044264       0.40037913297496985       0.70627168221540571       0.86823775809020642        0.0000000000000000     
   515.91937193257309       0.39566761265567957       0.70105546364878590       0.86547963232553904        0.0000000000000000     
   540.23390284308709       0.39103877089634997       0.69588476449113335       0.86267090505427502        0.0000000000000000     
   565.69434229195247       0.38649068447553181       0.69075946149043421       0.85981452990806462        0.0000000000000000     
   592.35469528478120       0.38202148576938549       0.68567941077357131       0.85691333506453060        0.0000000000000000     
   620.27151200468575       0.37762936083407439       0.68064444931677948       0.85397002865828775        0.0000000000000000     
   649.50400776280264       0.37331254756487980       0.67565439632922042       0.85098720394143801        0.0000000000000000     
   680.11418860190349       0.36906933392854541       0.67070905455481511       0.84796734420614683        0.0000000000000000     
   712.16698281952677       0.36489805626554950       0.66580821149716030       0.84491282748120711        0.0000000000000000     
   745.73037868960080       0.36079709765915291       0.66095164057204725       0.84182593101385206        0.0000000000000000     
   780.87556867468970       0.35676488636414683       0.65613910219184368       0.83870883554746944        0.0000000000000000     
   817.67710043474824       0.35279989431252584       0.65137034477816891       0.83556362939807594        0.0000000000000000     
   856.21303495270729       0.34890063565497170       0.64664510574200529       0.83239231236790756        0.0000000000000000     
   896.56511211228224       0.34506566537079281       0.64196311237161130       0.82919679947007674        0.0000000000000000     
   938.81892407922680       0.34129357792962678       0.63732408268150009       0.82597892447001442        0.0000000000000000     
   983.06409685378821       0.33758300600342539       0.63272772621389228       0.82274044329101204        0.0000000000000000     
   1029.3944803794759       0.33393261922718803       0.62817374478997690       0.81948303726116944        0.0000000000000000     
   1077.9083476113706       0.33034112300634422       0.62366183321634083       0.81620831621648615        0.0000000000000000     
   1128.7086029662387       0.32680725736878763       0.61919167994915103       0.81291782146698255        0.0000000000000000     
   1181.9030005965974       0.32332979585966487       0.61476296771851247       0.80961302863239437        0.0000000000000000     
   1237.6043729517164       0.31990754447710562       0.61037537411528298       0.80629535035362243        0.0000000000000000     
   1295.9308701103755       0.31653934064716926       0.60602857214249828       0.80296613888582491        0.0000000000000000     
   1357.0062103930161       0.31322405223637068       0.60172223073341868       0.79962668857872621        0.0000000000000000     
   1420.9599427848923       0.30996057660021148       0.59745601523811309       0.79627823824943100        0.0000000000000000     
   1487.9277217268332       0.30674783966623009       0.59322958788035596       0.79292197345276882        0.0000000000000000     
   1558.0515948565014       0.30358479505014363       0.58904260818653964       0.78955902865393557        0.0000000000000000     
   1631.4803043104764       0.30047042320372491       0.58489473338817921       0.78619048930796176        0.0000000000000000     
   1708.3696022262686       0.29740373059311798       0.58078561879951240       0.78281739385031091        0.0000000000000000     
   1788.8825811134850       0.29438374890635827       0.57671491817160792       0.77944073560269034        0.0000000000000000     
   1873.1900197949105       0.29140953428891331       0.57268228402429955       0.77606146459796710        0.0000000000000000     
   1961.4707456512799       0.28848016660612152       0.56868736795721631       0.77268048932786393        0.0000000000000000     
   2053.9120139381398       0.28559474873144930       0.56472982094107294       0.76929867841696331        0.0000000000000000     
   2150.7099049793446       0.28275240585954109       0.56080929359034937       0.76591686222634447        0.0000000000000000     
   2252.0697400797089       0.27995228484307999       0.55692543641839476       0.76253583439002826        0.0000000000000000     
   2358.2065170390315       0.27719355355251796       0.55307790007596114       0.75915635328725872        0.0000000000000000     
   2469.3453661912495       0.27447540025778239       0.54926633557409299       0.75577914345347952        0.0000000000000000     
   2585.7220279360617       0.27179703303109887       0.54549039449225412       0.75240489693274826        0.0000000000000000     
   2707.5833527759132       0.26915767917010958       0.54174972917253006       0.74903427457419280        0.0000000000000000     
   2835.1878249190304       0.26655658464050508       0.53804399290068361       0.74566790727498178        0.0000000000000000     
   2968.8061105590514       0.26399301353741589       0.53437284007481423       0.74230639717216973        0.0000000000000000     
   3108.7216319943373       0.26146624756484815       0.53073592636230926       0.73895031878566653        0.0000000000000000     
   3255.2311688046525       0.25897558553247474       0.52713290884576103       0.73560022011447090        0.0000000000000000     
   3408.6454873604480       0.25652034286912306       0.52356344615846229       0.73225662368820810        0.0000000000000000     
   3569.2899999999959       0.25409985115233114       0.52002719861007662       0.72892002757591223        0.0000000000000000     
