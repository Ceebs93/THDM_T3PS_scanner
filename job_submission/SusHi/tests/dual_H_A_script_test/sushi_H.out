# Date: 05.05.2019 at 13:44:46                     #
# For the reconstructed input file, check below.   #
#--------------------------------------------------#
# Please cite the following papers (for this run):
# Harlander:2012pb    
# Harlander:2016hcx   
# Harlander:2002wh    
# Harlander:2003ai    
# Aglietti:2004nj     
# Bonciani:2010ms     
# Eriksson:2009ws     
# Harlander:2005rq    
# Chetyrkin:2000yt    
#--------------------------------------------------#
Block SUSHIinfo
         1   1.6.1        # SusHi version
         5   1.7.0        # 2HDMC version
Block SUSHIggh # Bon appetit 
         1     8.85850591E-01   # ggh XS in pb                      
       101     2.03277445E-03   # +/- integ. error: ggh XS in pb                      
       102    -4.10959658E-02   # - from muR variation: ggh XS in pb                      
       103     2.22739021E-02   # + from muR variation: ggh XS in pb                      
# within [  0.50000000E+00,  0.20000000E+01]*SCALES(1)*MASSOUT(1).
Block SUSHIbbh # Bon appetit 
         1     6.19458550E-04   # bbh XS in pb                  
       101     9.48026914E-08   # +/- integr. error: bbh XS in pb                  
Block XSGGH # ggh xsec in pb (w/o EW) (+/- integ.err.)
         1     4.35848964E-01   # LO w/ NLO PDFs
         2     7.44126273E-01   # NLO
        21     7.72231754E-01   # NLO gg
        22    -2.81443036E-02   # NLO qg
        23     3.88223147E-05   # NLO qq
       101     2.46773132E-07   # +/-: LO w/ NLO PDFs
       102     7.52044104E-05   # +/-: NLO
       121     5.37085608E-05   # +/-: NLO gg
       122     5.26411494E-05   # +/-: NLO qg
       123     5.68766310E-08   # +/-: NLO qq
Block XSGGHEFF # ggh xsec in heavy top limit (from ggh@nnlo) (+/- integ.err.)
         1     8.29717314E-01   # NLO
       101     1.14088197E-04   # +/-: NLO
        19     8.19555239E-01   # NLO'
       191     1.08863811E-04   # +/-: NLO'
         2     9.71502896E-01   # NNLO
       102     2.02817678E-03   # +/-: NNLO
         4    -8.23303503E-05   # electroweak factor
Block XSBBH # bbh xsec in pb (+/- integ.err.)
         1     5.57948535E-04   # LO
       101     3.65376144E-09   # +/-: LO
         2     5.81055103E-04   # NLO
       102     3.22235713E-09   # +/-: NLO
         3     6.19458550E-04   # NNLO
       103     9.48026914E-08   # +/-: NNLO
Block HGGSUSY # couplings of 2. CP-even H to 3. gen.
       101     1.04793686E+00   # g_t^phi
       201    -9.38729076E-01   # g_b^phi
Block BBHREWEIGHT # top and bottom reweighting factors
         1     1.04793686E+00   # g_t
         2    -9.38729076E-01   # g_b
Block MASSOUT
         1     7.13858000E+02   # Mphi in GeV
         5     4.75000000E+00   # m_b(m_b), MSbar
         6     1.72500000E+02   # m_t(pole)
        23     9.11534900E+01   # m_Z
        24     8.03912615E+01   # m_W
        35     7.13858000E+02   # 2. CP-even Higgs mass in GeV
Block 2HDMCOUT # 2HDMC results for
         1    1   # stability, 0=failed, 1=ok
         2    0   # perturbativity, 0=failed, 1=ok
         3    0   # unitarity, 0=failed, 1=ok
         7     5.79698077E-03   # h width in GeV
         8     2.99120033E+02   # H width in GeV
         9     5.22315753E+01   # A width in GeV
Block INTERNALMASSES # Masses in GeV
        50     4.75000000E+00   # m_b(m_b), MSbar
        51     2.97899326E+00   # m_b(mu_R) MSbar
        52     4.75000000E+00   # m_b(pole)
        53     4.75000000E+00   # m_b used for internal masses
        54     4.75000000E+00   # m_b for bottom Yukawa
Block ALPHASOUT # values from LHAPDF set
         2     9.92925579E-02   # alpha_s(muRggh) @ NLO
        11     9.01468136E-02   # alpha_s(muRbbh)
         3     9.79024320E-02   # alpha_s(muRggh) @ NNLO
       101     1.35008398E-01   # alpha_s(Mz) @ LO  
       102     1.20006952E-01   # alpha_s(Mz) @ NLO 
       103     1.20006952E-01   # alpha_s(Mz) @ NNLO
#--------------------------------------------------#
# Reconstructed input file of this run             #
#--------------------------------------------------#
Block SUSHI
         1      2   # Chosen model: 0=SM, 1=MSSM, 2=2HDM, 3=NMSSM
         2     12   # 11=h, 12=H, 21=A              
         3      0   # Particle collider: 0=pp, 1=ppbar
         4     1.30000000E+04   # center-of-mass energy in GeV
         5      2   # Order for ggh
         6      2   # Order for bbh
         7      1   # Electroweak contributions to ggh
        11      1   # [1/0] do/not expand C1 perturbatively
        19      0   # 0 = silent mode of SusHi, 1 = normal output
        20      0   # ggh@nnlo subprocesses: 0=all, 10=ind. contributions
        21      0   # bbh@nnlo subprocesses: 0=all
Block GGHMT
        -1      3   # factor out exact LO result at LO(=0)/NLO(=1)/etc.
         0     -1   # expand through 1/mt^n at LO [-1=exact]
         1      0   # expand through 1/mt^n at NLO
        11      0   # expand gg through 1/mt^n at NLO
        12      0   # expand qg through 1/mt^n at NLO
        13      0   # expand qqbar through 1/mt^n at NLO
         2      0   # expand through 1/mt^n at NNLO
        21      0   # expand gg through 1/mt^n at NNLO
        22      0   # expand qg through 1/mt^n at NNLO
        23      0   # expand qqbar through 1/mt^n at NNLO
        24      0   # expand qq through 1/mt^n at NNLO
        25      0   # expand qqprime through 1/mt^n at NNLO
        10      0   # [0/1]: do not/match to high energy limit at NLO
        20      0   # [0/1]: do not/match to high energy limit at NNLO
        30      0   # [0/1]: do not/match to high energy limit at N3LO
Block GGHSOFT # parameters for soft expansion
         1    0  0  0   # NLO: [0/1=n/y] [order] sig(x)/x^[n]
         2    0  0  0   # NNLO: [0/1=n/y] [order] sig(x)/x^[n]
Block 2HDMC
         1      4   # parametrization of 2HDM
         2      2   # 2HDM type
        10      0   # [0/1] ignore validity checks [yes/no]
         3     1.10000000E+00   # tan(beta)
        31     1.25000000E+02   # mh
        32     7.13858000E+02   # mH
        33    -1.48993000E-01   # cos(beta-alpha)
        34     1.85180000E+00   # Z4
        35    -5.03961000E+00   # Z5
        36    -5.87636000E+00   # Z7
Block SMINPUTS
         1     1.27934000E+02   # 1/alpha_em(MZ) SM MSbar
         2     1.16637000E-05   # G_F
         3     1.19000000E-01   # alpha_s(m_Z)
         4     9.11534900E+01   # m_Z(pole)
         5     4.75000000E+00   # m_b(m_b)
         6     1.72500000E+02   # m_t(pole)
Block DISTRIB
         1      0   # total XS                      
Block SCALES
         1     5.00000000E-01   # Renormalization scale muR/mh
         2     5.00000000E-01   # Factorization scale muF/mh
       101     5.00E-01  2.00E+00   # min and max for muR scale uncertainty around SCALES(1)
       102     1.00E-01  1.00E+01   100   # min/max/steps for table output of muR variation
        11     1.00000000E+00   # Renormalization scale muR/mh for bbh
        12     2.50000000E-01   # Factorization scale muF/mh for bbh
Block RENORMBOT
         1      0   # mb used for bottom Yukawa
         4     4.75000000E+00   # Fixed m_b(pole) in GeV
Block PDFSPEC
         1   MMHT2014lo68cl.LHgrid                           # LO   PDF set
        11      0   # LO   PDF set number
         2   MMHT2014nlo68cl.LHgrid                          # NLO  PDF set
        12      0   # NLO  PDF set number
         3   MMHT2014nnlo68cl.LHgrid                         # NNLO PDF set
        13      0   # NNLO PDF set number
Block VEGAS
# parameters for NLO SusHi
         1    10000   # Number of points
         2        5   # Number of iterations
         3        0   # Output format of VEGAS integration
# parameters for ggh@nnlo:
         4     2000   # Number of points
         5        5   # Number of iterations
        14     5000   # Number of points in second run
        15        2   # Number of iterations in second run
         6        0   # Output format of VEGAS integration
# parameters for bbh@nnlo:
         7     2000   # Number of points
         8        5   # Number of iterations
        17     5000   # Number of points in second run
        18        2   # Number of iterations in second run
         9        0   # Output format of VEGAS integration
Block FACTORS
         1     0.00000000E+00   # Factor multiplied with Yukawa of c quark
         2     1.00000000E+00   # t quark
         3     1.00000000E+00   # b quark
#--------------------------------------------------#
# End of SusHi output                              #
#--------------------------------------------------#
